docs/lifecycle/lfn_entry/opx.md

```diff
@@ -1,2 +1,287 @@
+    * Inocybe: DHCP Relay, monitoring/telemetry software, and other small contributions.
+    * Broadcom: monitoring and telemetry software/drivers.
+
+### Project Functionality
+#### Summary of candidate project functionality 
+
+OPX is an open source networking operating system. It provides an API for applications to work with platform and networking elements of a network switch, and it leverages the OCP’s Switch Abstraction Interface (SAI) to provide portability between various ASICs. Lastly, OPX runs on an unmodified Linux kernel in order to provide a familiar environment to users and an easy path to onboard new platforms and applications.
+
+#### Summary of candidate project technology components and purposes – 
+
+The [OPX architecture](https://github.com/open-switch/opx-docs/wiki/Architecture) consists of three major components:
+
+* [Platform Adaptation Services](https://github.com/open-switch/opx-docs/wiki/Architecture#platform-adaptation-service) (PAS) – Provides system integrators with a common way to define platform elements like buses, fans, power supplies, LEDs, supported media, etc.
+
+* [Network Adaptation Services](https://github.com/open-switch/opx-docs/wiki/Network-adaptation-service) (NAS) – Working alongside SAI (and a vendor’s implementation of SAI), provides and manages the high-level Network Processing Unit (NPU) abstraction and adaptation. NAS abstracts and aggregates the core functionality required for networking access at Layer-1, Layer-2, and Layer-3, ACL, QoS, and network monitoring. NAS is also responsible for providing packet I/O services using the Linux kernel IP stack and Netlink interface.
+
+* [Control Plane Services](https://github.com/open-switch/opx-docs/wiki/Architecture#control-plane-services) (CPS) – Provides an object-centric framework that mediates interactions between applications and the NAS and PAS services. Client applications execute create/set/get/delete operations on individual objects or lists of objects, and services handle those operations. CPS also supports a publisher/subscriber model, where clients can register for events generated when objects are created, modified, or deleted. The publisher/subscriber approach and object-centric operations allow for completely independent operation of client and server applications.
+
+Summary of where candidate project complements functionality already provided by project(s) within LFN
+
+Since OPX runs on an unmodified Linux kernel and enables a lot of networking functionality with the Netlink API, many projects that work with Linux can similarly work with OPX. One such example is FRRouting, which provides a Layer-3 networking control plane – as FRRouting installs routes into the Linux kernel, those routes are also populated in the switch’s networking ASIC (via the NAS Linux adapter) so that the switch can forward traffic accordingly.
+
+Summary of where candidate project overlaps functionality already provided by project(s) within LFN
```

Comment:
clear, is there any plan to find synergies?

what link with DANOS if any,  https://www.linuxfoundation.org/networking-orchestration/2018/03/the-linux-foundation-hosts-danos-project-a-unified-network-operating-system/